src/client/styles/globals.css

```diff
@@ -195,4 +282,31 @@ table {
   .header-bottom {
     padding: 0 20px;
   }
+}
+
+/* Home Page */
+
+.tour,
+.info {
+  background-image: url(src/client/styles/img/vintageWood.webp);
+}
+
+.tour {
+  height: 40vh;
+}
+
+.info {
+  height: 60vh;
+}
+
+.divider {
+  transform: scale(.3);
+}
+
+.brooks {
+  transform: scale(.8);
+}
+
+.brooks-stream {
+  transform: scale(.4);
```

Comment:
instead of transform to shrink the picture it should have a max-width
max-width: 24rem should be fine here.